api/graphql/schema.graphql

```diff
@@ -1376,6 +1377,22 @@ input UpdateTeamInput {
   contactEmail: Email @rename(attribute: "contact_email")
 }
 
+input CreateCommunityInput {
+  key: String!
+    @rules(
+      apply: ["unique:communities,key"]
+      messages: [{ rule: "unique", message: "CommunityKeyInUse" }]
+    )
+  name: LocalizedStringInput
+  description: LocalizedStringInput
+}
```

Comment:
Looks like you created a validator for this so you should be able to apply the `@validator` directive to the mutation and not need the `@rules` here.

Reply:
df48f71f8cecf9299951b7b5f5d20ac04c543b20